Latex/inhalt/Abstract_Freigabeerklärung_Bachelorthesis.tex

```diff
@@ -0,0 +1,91 @@
+\cleardoublepage
+\section{Abstract zur Bachelorthesis}
+    % Alternativ zu Vektorgrafik-Logos (SVG) werden Befehle zur Nutzung von Pixelgrafik-Logos (z.B. PNG) bereitgestellt,
+    % falls Inkscape nicht installiert werden, bzw. nicht der PATH-Variable hinzugefügt werden soll.
+    %! Die SVG-Version sieht im Druck deutlich besser aus.
+    \begin{minipage}{0.5\columnwidth}
+        \includesvg[width=\columnwidth]{bilder/ba_glauchau_logo.svg}
+        %\includegraphics[width=\columnwidth]{bilder/ba-gc-logo.png}
+    \end{minipage}
+    \begin{minipage}{0.45\columnwidth}
+        \begin{flushright}
+            {\small nach 4BA-F.258\\}
+            %! hier Firmenlogo aktivieren!
+            %\includesvg[height=10mm,inkscapelatex=false]{bilder/firmenlogo.svg}
+            %\includegraphics[height=10mm]{bilder/firmenlogo.png}
+        \end{flushright}
+    \end{minipage}
+    \par\noindent\rule{\columnwidth}{.5pt}
+
+    \textbf{\Large{Abstract zur Bachelorthesis}}
+    \begin{table}[H]
+        \centering
+        \fontsize{14pt}{15pt}
+        \begin{tabularx}{\columnwidth}{p{3.3cm} X}
+            \textbf{Studiengang:}               & \studiengang \\
+            \textbf{Name:}                      & \autoreins \\
+            \textbf{Thema:}                     & \titel \\
+            \textbf{Jahr:}                      & \jahr \\
+            \multirow{2}{*}{\textbf{Betreuer:}} & \betreuereins\hspace*{2cm}(\institutioneins) \\
+                                                & \betreuerzwei\hspace*{2cm}(\institutionzwei) \\
```

Comment:
wie @DSczyrba schon meinte, sieht das in gerendert echt kacke aus

Reply:
Scheint so gewollt zu sein :D

Reply:
halte ich für unwahrscheinlich. Vgl
```
typ 1: vorname name (Firma)
typ 2: vorname langer nachname (Staatliche BA dings)
```
```
typ 1: vorname name                (Firma)
typ 2: vorname langer nachname                (Staatliche BA dings)
```
jedenfalls versteh ich so den statischen hspace

Reply:
naja wie würdest du tun... in dem dokument is ein etwas längerer Abstand zwischen den Namen und Institutionen

Reply:
![grafik](https://user-images.githubusercontent.com/24256864/130799700-b6f73084-dc00-4676-bc1a-f04ce8705323.png)
alternativ auffüllen, das sieht aber so aus

Reply:
ich würde da eines der folgenden:
* keinen abestand hin packen (einfach nur ein leerzeichen, dann die klammer)
* die klammern inklusive text rechtsbündig machen (sieht wahrscheinlich blöd aus wenn der name links und dir org rechts steht)
* die klammern miteinander alignen (fangen in der gleichen spalte an) -> immernoch komischer abstand bei 1 von beiden, das andere wie option 1
* die institutionen ähnlich zur titelseite in eigene zeilen

Reply:
![grafik](https://user-images.githubusercontent.com/24256864/130799886-97ff14cf-612c-4930-a0dc-4db14d206efa.png)
wobei im original siehts auf den zweiten blick aus, als wäre die zelle geteilt... die frage is: wie mach ich das in dynamisch?

Reply:
ist auch ne option, wenn auch nicht die einfachste. wie genau das wird ist mir egal, ich fänd's bloss cool wenn
* entweder beide klammern direkt hinter den namen stehen
* oder jeweils die öffnenden (oder notfalls schliessenden) klammern miteinander aligned sind

Reply:
okay ich habs, kommt gleich n neuer pr auf dev

Reply:
siehe #111